.travis.yml

```diff
@@ -0,0 +1,16 @@
+dist: xenial
+language: php
+php:
+  - '7.1'
```

Comment:
We should also test for 7.0, because this will be the minimum requirement for v3.0.